source/configure/calls-troubleshooting.md

```diff
@@ -0,0 +1,385 @@
+     ```
+
+3. **API connectivity**:
+   - Verify that Mattermost servers can reach the RTCD API endpoint
+   - Check that the API key is correctly configured in both Mattermost and RTCD
+
+4. **Plugin configuration**:
+   - Ensure the Calls plugin is enabled and properly configured
+   - Verify the RTCD service URL is correct in the System Console
+
+### Audio Issues
+
+**Symptoms**: Users can connect to calls, but audio is one-way, choppy, or not working.
+
+**Possible causes and solutions**:
+
+1. **Client permissions**:
+   - Ensure browser/app has microphone permissions
+   - Check if users are using multiple audio devices that might interfere
+
+2. **Network quality**:
+   - High latency or packet loss can cause audio issues
+   - Try testing with TCP fallback enabled (requires RTCD v0.11+ and Calls v0.17+)
+
+3. **Audio device configuration**:
+   - Users should verify their audio input/output settings
+   - Try different browsers or the desktop app
+
+### Call Quality Issues
+
+**Symptoms**: Calls connect but quality is poor, with latency, echo, or distortion.
+
+**Possible causes and solutions**:
+
+1. **Server resources**:
+   - Check CPU usage on RTCD servers - high CPU can cause quality issues
+   - Refer to the [Calls Metrics and Monitoring](calls-metrics-monitoring.md) guide for detailed instructions on monitoring and optimizing performance
+   - Monitor network bandwidth usage
+
+2. **Network congestion**:
+   - Check for packet loss between clients and RTCD
+   - Consider network QoS settings to prioritize real-time traffic
+
+3. **Client-side issues**:
+   - Browser or app limitations
+   - Hardware limitations (CPU, memory)
+   - Network congestion at the user's location
+
+## Connectivity Troubleshooting
+
+### Basic Connectivity Tests
+
+1. **HTTP API connectivity test**:
+
+   Test if the RTCD API is reachable:
+
+   ```bash
+   curl http://YOUR_RTCD_SERVER:8045/version
+   # Example response: {"buildDate":"2025-04-02 21:33","buildVersion":"v1.1.0","buildHash":"7bc1f7a","goVersion":"go1.23.6","goOS":"linux","goArch":"amd64"}   ```
+
+2. **UDP connectivity test**:
+
+   On the RTCD server:
+   
+   ```bash
+   nc -l -u -p 8443
+   ```
+
+   On a client machine:
+   
+   ```bash
+   nc -v -u YOUR_RTCD_SERVER 8443
+   ```
+      
+   Type a message and press Enter. If you see the message on both sides, UDP connectivity is working.
+
+3. **TCP fallback connectivity test**:
+
+   Same as the UDP test, but without the `-u` flag:
+   
+   On the RTCD server:
+   
+   ```bash
+   nc -l -p 8443
+   ```
+
+   On a client machine:
+   
+   ```bash
+   nc -v YOUR_RTCD_SERVER 8443
+   ```
+
+### Network Packet Analysis
+
+To capture and analyze network traffic:
+
+1. **Capture UDP traffic on the RTCD server**:
+
+   ```bash
+   sudo tcpdump -n 'udp port 8443' -i any
+   ```
+
+2. **Capture TCP API traffic**:
+
+   ```bash
+   sudo tcpdump -n 'tcp port 8045' -i any
+   ```
```

Comment:
This feels unnecessary as I don't think anyone would need to go to this level of debugging for a simple HTTP API.